### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by tescher** [BanklessDAO/bankless-dao-ghost-website#124]
Multiple accessibility failures including color contrast and screen reader support. Fix issues identified by the accessibility check: https://www.accessibilitychecker.org/audit/?website=https%3A%2F%2Fbankless-dao-ghost-website.vercel.app%2F&flag=us&has_utm_source=false

[Transferred from the Website QA Spreadsheet]
